models/iaf_wang_2002.h

```diff
@@ -0,0 +1,518 @@
+
+* an approximate version of the neuron model described in [1]_.
+* exponential conductance-based AMPA and GABA-synapses
+* exponential conductance-based NMDA-synapses weighted such that it approximates the original non-linear dynamics
+* a fixed refractory period
+* no adaptation mechanisms
+
+Neuron and synaptic dynamics
+..................................................
+
+The membrane potential and synaptic variables evolve according to
+
+.. math::
+
+    C_\mathrm{m} \frac{dV(t)}{dt} &= -g_\mathrm{L} (V(t) - V_\mathrm{L}) - I_\mathrm{syn} (t) \\[3ex]
+    I_\mathrm{syn}(t) &= I_\mathrm{AMPA}(t) + I_\mathrm{NMDA}(t) + I_\mathrm{GABA}(t) (t) \\[3ex]
+    I_\mathrm{AMPA} &= (V(t) - V_E)\sum_{j \in \Gamma_\mathrm{ex}}^{N_E}w_jS_{j,\mathrm{AMPA}}(t) \\[3ex]
+    I_\mathrm{NMDA} &= \frac{(V(t) - V_E)}{1+[\mathrm{Mg^{2+}}]\mathrm{exp}(-0.062V(t))/3.57}\sum_{j \in
+\Gamma_\mathrm{ex}}^{N_E}w_jS_{j,\mathrm{NMDA}}(t) \\[3ex] I_\mathrm{GABA} &= (V(t) - V_I)\sum_{j \in
+\Gamma_\mathrm{in}}^{N_E}w_jS_{j,\mathrm{GABA}}(t) \\[5ex] \frac{dS_{j,\mathrm{AMPA}}}{dt} &=
+-\frac{j,S_{\mathrm{AMPA}}}{\tau_\mathrm{AMPA}}+\sum_{k \in \Delta_j} \delta (t - t_j^k) \\[3ex]
+    \frac{dS_{j,\mathrm{GABA}}}{dt} &= -\frac{S_{j,\mathrm{GABA}}}{\tau_\mathrm{GABA}} + \sum_{k \in \Delta_j} \delta (t
+- t_j^k) \\[3ex] \frac{dS_{j,\mathrm{NMDA}}}{dt} &= -\frac{S_{j,\mathrm{NMDA}}}{\tau_\mathrm{NMDA,decay}} + \sum_{k \in
+\Delta_j} (k_0 + k_1 S(t)) \delta (t - t_j^k) \\[3ex]
+
+where :math:`\Gamma_\mathrm{ex}` and :math:`\Gamma_\mathrm{in}` are index sets for presynaptic excitatory and inhibitory
+neurons respectively, and :math:`\Delta_j` is an index set for the spike times of neuron :math:`j`.
+
+.. math::
+
+    k_0 &= \mathrm{exp}(-\alpha \tau_\mathrm{r}) - 1 \\[3ex]
+    k_1 &= -\alpha \tau_\mathrm{r} \mathrm{E_N} \Big[ \frac{\tau_\mathrm{r}}{\tau_\mathrm{d}}, \alpha \tau_\mathrm{r}
+\Big] + (\alpha \tau_\mathrm{r})^{\frac{\tau_\mathrm{r}}{\tau_\mathrm{d}}} \Gamma \Big[ 1 -
+\frac{\tau_\mathrm{r}}{\tau_\mathrm{d}} \Big]
+
+where :math:`\mathrm{E_N}` is the `generalized exponential integral
+<https://en.wikipedia.org/wiki/Exponential_integral#Generalization>`_, and :math:`\Gamma` is the `gamma function
+<https://en.wikipedia.org/wiki/Gamma_function>`_. For these values of :math:`k_0` and :math:`k_1`, the approximate model
+will approach the exact model for large t.
+
+The specification of this model differs slightly from the one in [1]_. The parameters :math:`g_\mathrm{AMPA}`,
+:math:`g_\mathrm{GABA}`, and :math:`g_\mathrm{NMDA}` have been absorbed into the respective synaptic weights.
+Additionally, the synapses from the external population is not separated from the recurrent AMPA-synapses.
+
+For more implementation details and a comparison to the exact version see:
+
+- `wong_approximate_implementation <../model_details/wong_approximate_implementation.ipynb>`_
+
+
+Parameters
+++++++++++
+
+The following parameters can be set in the status dictionary.
+
+
```

Comment:
```suggestion
```

Could you format the table as in https://nest-simulator.readthedocs.io/en/stable/models/iaf_psc_alpha.html ?